src/pages/install/windows.md

```diff
@@ -0,0 +1,108 @@
+
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+<hgroup>
+
+← [Back to install instructions](/install)
+
+# Installing RGBDS on Windows
+
+</hgroup>
+
+The install instructions are a bit different depending on the environment in which you wish to use RGBDS.
+
+<Tabs>
+<TabItem value="wsl" label="WSL">
+
+For these, please refer to [the install instructions](linux) for your WSL's running Linux distribution.
+You can check what your WSL distribution is by running `wsl -l -v` in the Command Prompt.
+(The default Linux distribution on WSL is Ubuntu, whose package manager is `apt`.)
+
+</TabItem>
+<TabItem value="cygwin" label="Cygwin / MSYS2">
+
+1. Click the link of the version you want [from the list](/docs).
+   If you want to use [`master`](/docs/master#master) instead of a released version, [follow the corresponding instructions](/install/master).
```

Comment:
This may be confusing, we have a page for installing master, yet we keep repeating how to install master in unrelated places as well